app-accessibility/brltty/brltty-6.1.ebuild

```diff
@@ -56,15 +59,19 @@ DEPEND="
 		x11-libs/libXfixes
 		x11-libs/libXt
 		x11-libs/libXtst
-	)"
-RDEPEND="${DEPEND}
+	)
+"
+
+RDEPEND="
+	${DEPEND}
 	java? ( >=virtual/jre-1.4 )
 "
+
 BDEPEND="
-	virtual/pkgconfig
 	java? ( >=virtual/jdk-1.4 )
 	nls? ( virtual/libintl )
 	python? ( >=dev-python/cython-0.16[${PYTHON_USEDEP}] )
+	virtual/pkgconfig
```

Comment:
Normally, we put all unconditional then conditional, so it was originally OK.

Reply:
Dropped commit